src/components/TopProject.styles.ts

```diff
@@ -0,0 +1,255 @@
+import styled from "styled-components/macro";
+
+interface TopProjectProps {
+  isVisible?: boolean;
+}
+
+export const TopProjectContainer = styled.div`
+  border-radius: 20px;
+  position: relative;
+  justify-content: space-between;
+  width: 100%;
+  padding-top: 56.25%; /* 16:9 Aspect Ratio */
+  background: white;
+  padding-top: 35%;
+  margin-bottom: 2%;
+
+  @media (max-width: 900px) {
+    width: 50%;
+    height:250px;
+    margin: 0 auto;
+  }
+  @media (max-width: 660px) {
+    width: 60%;
+    height:280px;
+  }
+  @media (max-width: 500px) {
+    width: 70%;
+    height:270px;
+  }
+  @media (max-width: 400px) {
+    width: 80%;
+  }
+ 
+`;
+
+export const TopProjectTitle = styled.div<TopProjectProps>`
+  position: absolute;
+  text-align: left;
+  font-size: 2rem;
+  font-weight: bold;
+  color: black;
+  top: 20%;
+  left: 10%;
+  right: 10%;
+  transition: 0.3s ease-in-out;
+
+  @media (min-width: 900px) and (max-width:1100px) {
+       top: 9%;
+  }
+   @media (max-width: 900px) {
+    font-size: 1.5rem;
+    top: 10%;
+  }
+  @media (min-width: 250px) and (max-width:320px) {
+     font-size: 1.5rem;
+  }
+  
+`;
+
+export const TopProjectYear = styled.div<TopProjectProps>`
+  position: absolute;
+  text-align: left;
+  font-style: italic;
+  font-size: 1rem;
+  color: black;
+  top: 33%;
+  left: 10%;
+  right: 10%;
+  transition: 0.3s ease-in-out;
+
+  @media (min-width: 900px) and (max-width:1100px) {
+       top: 25%;
+  }
+   @media (max-width: 900px) {
+    font-size: 1rem;
+    top: 18%;
+  }
+  @media (min-width: 250px) and (max-width:320px) {
+     font-size: 0.8rem;
+     top: 19%;
+  }
+ 
+`;
+
+export const TopProjectContent = styled.div<TopProjectProps>`
+  /* We can change this, but position should be absolute still */
+  position: absolute;
+  text-align: left;
+  font-size: 1rem;
+  color: black;
+  top: 45%;
+  left: 10%;
+  right: 15%; 
+  width: 35%;
+  transition: 0.3s ease-in-out;
+
+   @media (min-width: 900px) and (max-width:930px) {
+        font-size: 0.8rem;
+  }
+    @media (max-width: 900px) {
+    font-size: 0.9rem;
+    top: 25%;
+    width: 90%;
+  }
+   @media (max-width: 660px) {
+    font-size: 0.8rem;
+  }
+   @media (max-width: 500px) {
+    font-size: 0.7rem;
+  }
+ 
+`;
+
+export const TopProjectButton = styled.a`
```

Comment:
I would advise against creating a button from scratch with all these overly specific css rules. For consistency, let's use our `HoverButton` in the featured project card, and you can look into adding a github emoji in the text for the button to achieve a similar look.